dev/mima

```diff
@@ -24,9 +24,9 @@ set -e
 FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
 cd "$FWDIR"
 
-SPARK_PROFILES=${1:-"-Pscala-2.13 -Pmesos -Pkubernetes -Pyarn -Pspark-ganglia-lgpl -Pkinesis-asl -Phive-thriftserver -Phive"}
-TOOLS_CLASSPATH="$(build/sbt -Pscala-2.13 -DcopyDependencies=false "export tools/fullClasspath" | grep jar | tail -n1)"
-OLD_DEPS_CLASSPATH="$(build/sbt -Pscala-2.13 -DcopyDependencies=false $SPARK_PROFILES "export oldDeps/fullClasspath" | grep jar | tail -n1)"
```

Comment:
This is good. What I conceded is only `pom.xml` file changes, @HyukjinKwon .

Reply:
While cleaning this up, I suggest removing `2.13` from `VALID_VERSIONS` in the `change-scala-version.sh` script, otherwise if we execute `dev/change-scala-version.sh 2.13`, these files will generate some unnecessary git diff.

https://github.com/apache/spark/blame/5f643ee1444d41ca78ef397d9675efac550a6967/dev/change-scala-version.sh#L22

Reply:
I commented `dev/change-scala-version.sh` usage out in the README.md :-).